README.md

```diff
@@ -1,45 +1,146 @@
 # Catapult-explorer
 
-## Setup Client
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+A catapult explorer is a browser for the NEM2 blockchain, user can use it to view and track transactions, accounts, assets, and blocks.
+
+## Architecture
+
+In Catapult Explorer, `Client` is handle all fronted development such as UI / UX. `Server` is handle API / SDK request from Catapult Network.
```

Comment:
The catapult-server is composed of two separate modules:

* `client`: Handles the UI / UX of the explorer. 
* `server` Handles the API / SDK request from Catapult Network.